slowzones.py

```diff
@@ -46,22 +46,29 @@
 def main():
     slow_zones = requests.get("https://dashboard.transitmatters.org/static/slowzones/all_slow.json")
 
-    grouped_sz_today = generate_grouped_slow_zone_list(slow_zones.json(), date.today())
+    if slow_zones.json()['updated_on'] != date.today().strftime("%Y-%m-%d"):
```

Comment:
Can we check the dates themselves rather than comparing string representations? Even if it works, it's generally bad practice

https://stackoverflow.com/a/6407393